### PR TITLE
Rek reduxify feedback

### PR DESCRIPTION
**Pull request by kilbergr**
This PR includes the duck with all reduxified code related to Submitted Feedback. It includes very basic tests. It also includes basic tests for issueCards, which had to be fleshed out a bit.

### CODE REVIEW

File: src/scenes/SubmittedFeedback/IssueCards.test.js [hunk 1]
```diff
@@ -3,9 +3,29 @@ import ReactDOM from 'react-dom';
...
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<IssueCards issues={null} />, div);
+describe('Null Issues on IssueCards', () => {
+  const issues = null;
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<IssueCards issues={issues} />, div);
```
Comment (tinyels) [inline src/scenes/SubmittedFeedback/IssueCards.test.js]: Is there a reason not to switch to `shallow` or  `mount` here?

File: src/scenes/SubmittedFeedback/IssueCards.test.js [hunk 2]
```diff
@@ -3,9 +3,29 @@ import ReactDOM from 'react-dom';
...
+  });
 });
 
-//todo: test that variations of issues (null,empty,data) render as expected
+describe('Issues on IssueCards', () => {
+  const issues = ['Too few dogs.'];
```
Comment (tinyels) [inline src/scenes/SubmittedFeedback/IssueCards.test.js]: I think the code is expecting the array to have objects of a particular shape. I should have written a proptype for this specifying the shape.
The shape it is expecting is  
```
{
    "id": PropTypes.string.isRequired,
    "description": PropTypes.string.isRequired,
}
```

File: src/scenes/SubmittedFeedback/ducks.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    let action;
+    store.dispatch(createShowIssuesRequest());
+    store.dispatch(createShowIssuesSuccess());
+    store.dispatch(createShowIssuesFailure());
+    action = store.getActions();
+    expect(action[0].type).toBe('SHOW_ISSUES');
+    expect(action[1].type).toBe('SHOW_ISSUES_SUCCESS');
+    expect(action[2].type).toBe('SHOW_ISSUES_FAILURE');
+  });
+});
```
Comment (tinyels) [inline src/scenes/SubmittedFeedback/ducks.test.js]: Do we want to add tests for the action creators? That would require figuring out how to mock the async api calls.

Comment (tinyels) [inline src/scenes/SubmittedFeedback/ducks.test.js]: Do we want to add tests for the action creators? That would require figuring out how to mock the async api calls.

File: src/scenes/SubmittedFeedback/IssueCards.test.js (hunk 2 above)
Comment (kilbergr) [reply src/scenes/SubmittedFeedback/IssueCards.test.js]: Ohh I was trying to do with {"key", "issue"}...

File: src/scenes/SubmittedFeedback/IssueCards.test.js (hunk 1 above)
Comment (kilbergr) [reply src/scenes/SubmittedFeedback/IssueCards.test.js]: Nope. Done.

File: src/scenes/SubmittedFeedback/ducks.test.js (hunk 3 above)
Comment (kilbergr) [reply src/scenes/SubmittedFeedback/ducks.test.js]: Created pivotal tracker task for this: https://www.pivotaltracker.com/story/show/154860395

**Pull request merged**